### ISSUE TITLE
Memory leak in fragment when use Checkout.setResultCallback

### ISSUE THREAD

**Issue by chim0380** [yandex-money/yandex-checkout-android-sdk#13]
В вашем SDK  есть метод **Checkout.setResultCallback** , он не принимает параметр null и нет методов по отписке от callback, и из-за этого происходит утечка памяти в том фрагменте приложения, в котором используется Checkout

Checkout.detach() так же не отписывает от callback

Утечка легко обнаруживается при помощи **LeakCanary**

Если не использовать setResultCallback , то утечки нет.

Было бы здорового, если бы в следующей версии, можно было бы отписываться от Callback.

**Comment by drukarev** [comment]
Спасибо, исправим в ближайшей версии.

**Comment by drukarev** [comment]
Fixed in 1.3.0.5
